tests/sles4sap/qesapdeployment/configure.pm

```diff
@@ -56,6 +56,11 @@ sub run {
         $variables{HANA_DATA_DISK_TYPE} = get_var("QESAPDEPLOY_HANA_DISK_TYPE", "pd-ssd");
         $variables{HANA_LOG_DISK_TYPE} = get_var("QESAPDEPLOY_HANA_DISK_TYPE", "pd-ssd");
     }
+
+    my %peering_settings = qesap_calculate_az_address_range(slot => get_var("WORKER_INSTANCE"));
```

Comment:
Do we want to use `WORKER_INSTANCE` or `WORKER_ID`?

For example:

Job https://openqa.suse.de/tests/11052344 ran in worker `worker11:5` while job https://openqa.suse.de/tests/11049678 ran in worker `grenache-1:5`.

Both jobs have `"WORKER_INSTANCE" : 5`, but, first job has `"WORKER_ID" : 2250` and second one has `"WORKER_ID" : 1106`.

Reply:
To be checked here too https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17020/files#diff-d537e7676e4de934957a302dd9662d07237c473a02581e3fa2ca4065a6471c6dR153